### PR TITLE
8245268: -Xcomp is missing from java launcher documentation

### PR DESCRIPTION
**Pull request by veresov**
Updated man pages from markdown sources.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8245268](https://bugs.openjdk.org/browse/JDK-8245268): -Xcomp is missing from java launcher documentation

### Reviewers
 * [Vladimir Kozlov](https://openjdk.org/census#kvn) (@vnkozlov - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk19 pull/103/head:pull/103` \
`$ git checkout pull/103`

Update a local copy of the PR: \
`$ git checkout pull/103` \
`$ git pull https://git.openjdk.org/jdk19 pull/103/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 103`

View PR using the GUI difftool: \
`$ git pr show -t 103`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk19/pull/103.diff">https://git.openjdk.org/jdk19/pull/103.diff</a>

</details>

### CODE REVIEW

File: src/java.base/share/man/java.1 [hunk 1]
```diff
@@ -5355,7 +5364,7 @@ loaded on top of those in the \f[CB]<static_archive>\f[R].
 On Windows, the above path delimiter \f[CB]:\f[R] should be replaced with
 \f[CB];\f[R]
 .PP
-(The names "static" and "dynamic" are used for historical reasons.
+(The names "static" and "dyanmic" are used for historical reasons.
```
Comment (vnkozlov) [inline src/java.base/share/man/java.1:5367]: `dyanmic` ?

**Review by vnkozlov — COMMENTED** [review]
I am not sure some words changes are correct?

May be we should cleanup and update all these man pages as separate issue. And apply only `-Xcomp` changes.

@iklan you recently pushed changes #9024 which included only CDS related update.
Did you discuss this issue during your changes review?

**Comment by veresov** [comment]
> I am not sure some words changes are correct?
> 
> May be we should cleanup and update all these man pages as separate issue. And apply only `-Xcomp` changes.
> 

Then I'd have to manually craft a patch instead of actually generating them. It seems like it's all been out of sync for a long time.

**Comment by veresov** [comment]
There is also a question of which version is actually correct. Yeah, may be I should manually craft a patch just for the Xcomp part.

**Comment by iklam** [comment]
> There is also a question of which version is actually correct. Yeah, may be I should manually craft a patch just for the Xcomp part.

@veresov  please see the closed issue https://bugs.openjdk.org/browse/JDK-8287821

@vnkozlov when I did https://github.com/openjdk/jdk/pull/9024, I generated `java.1` from the `java.md` file. I then use the `meld` program on Linux to revert all changes in the `java.1` file that were unrelated to my changes in `java.md`. Yes, it was a pain.

**Comment by veresov** [comment]
Ok, I just hacked a patch file.

**Review by vnkozlov — APPROVED** [review]
Good

**Comment by veresov** [comment]
Thanks, Vladimir!
/integrate

**Pull request closed without merging**